src/modules/auth/controllers/meemoo.controller.ts

```diff
@@ -113,11 +113,14 @@ export class MeemooController {
 					Idp.MEEMOO,
 					ldapUser.attributes.entryUUID[0]
 				);
-				await this.collectionsService.create({
-					is_default: true,
-					user_profile_id: archiefUser.id,
-					name: i18n.t('modules/collections/controllers___default-collection-name'),
-				});
+				await this.collectionsService.create(
+					{
+						is_default: true,
+						user_profile_id: archiefUser.id,
+						name: i18n.t('modules/collections/controllers___default-collection-name'),
+					},
+					null // referer not important here
```

Comment:
Waarom kan `null` hier assigned worden aan een `string` functie-param?

Reply:
null kan je toch altijd voor alles gebruiken? Leek me hier logischer dan '', maar is persoonlijke voorkeur wss